__tests__/src/core/coords.spec.js

```diff
@@ -0,0 +1,37 @@
+import Coords from '../../../src/core/coords';
+
+describe('Game', () => {
+    describe('Coords', () => {
+        test('should construct coords wtesth default value', () => {
+            const coords = new Coords();
+
+            expect(coords.y).toEqual(0);
+            expect(coords.x).toEqual(0);
+        });
+
+        test('should return true if two coords are equals', () => {
+            const coords = new Coords(1, 3);
+            const coords2 = new Coords(1, 3);
+
+            expect(coords.equalsTo(coords2)).toBe(true);
```

Comment:
What's the difference between `expect().toEqual()` and `expect(a.equalsTo(b)).toBe(true)` ? Why are there both ?

Reply:
`equalsTo` is the tested method.
And I test the returned value of `a.equalsTo(b)`